libs/datasets/sources/hhs_hospital_dataset.py

```diff
@@ -1,24 +1,119 @@
+import dataclasses
 from functools import lru_cache
+from libs import pipeline
+
 from covidactnow.datapublic.common_fields import CommonFields
+import pandas as pd
+from libs.datasets.sources import can_scraper_helpers as ccd_helpers
 from libs.datasets import data_source
 from libs.datasets.timeseries import MultiRegionDataset
 
 
-class HHSHospitalDataset(data_source.DataSource):
-    SOURCE_TYPE = "HHSHospital"
+# Early data is noisy due to lack of reporting, etc. so we just trim it off.
+DEFAULT_START_DATE = "2020-09-01"
+CUSTOM_START_DATES = {
+    "02": "2020-10-06",  # Alaska
+    "04": "2020-09-02",  # Arizona
+    "15": "2020-10-10",  # Hawaii
+    "16": "2020-10-17",  # Idaho
+    "19": "2020-09-05",  # Iowa
+    "21": "2020-10-15",  # Kentucky
+    "28": "2020-11-11",  # Mississippi
+    "34": "2020-09-01",  # New Jersey
+    "38": "2020-11-03",  # North Dakota
+    "46": "2020-11-02",  # South Dakota
+    "47": "2020-09-20",  # Tennessee
+    "53": "2020-10-25",  # Washington
+}
+
+# Mapping of CAN scraper fields to common fields.
```

Comment:
Why refer to ScraperVariable.variable_name as "CAN scraper fields"? "CAN scraper variable_name" is more consistent. Same for the local var name in make_hhs_variable.